app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  has_many :votes
+  has_many :works, through: :votes
+
+  # I added this before I decided to do a current user path. Do I need this, then?
+  def self.current(session)
```

Comment:
From our spelunking: This method is only being used in the application layout file... which could be refactored to not use this method (called with `User.current`